keps/sig-api-machinery/4008-crd-ratcheting/README.md

```diff
@@ -824,14 +823,8 @@ For Beta and GA, add links to added tests together with links to k8s-triage for
 https://storage.googleapis.com/k8s-triage/index.html
 -->
 
-1. Show normal creation path is unaffected
-2. Tests that ratchet different classes of validations:
-    - JSON-Schema fields (minItems, dependencies, etc.)
-    - CRD-Validation Rules
-    - ListSets/Maps
-3. Tests that certain classes of valdiations are not ratcheted:
-    - Nested Validations (not, oneOf, allOf, anyOf)
-    - Transition Rules
+- [k8s.io/apiextensions-apiserver/test/integration.TestRatchetingFunctionality](https://github.com/kubernetes/kubernetes/blob/3ee81787685e47a7a5da22423c8ca4455577ecb3/staging/src/k8s.io/apiextensions-apiserver/test/integration/ratcheting_test.go#L428)
```

Comment:
Comment asks for links to the test in https://storage.googleapis.com/k8s-triage/index.html but Im ashamed to admit I'm not sure the proper way to generate a link that vends all failures for the tests. ex. searching `Ratcheting` [in a large date range](https://storage.googleapis.com/k8s-triage/index.html?date=2022-08-11&test=Ratcheting) returned no failures.